src/components/MotionGenerator.tsx

```diff
@@ -49,24 +62,10 @@ const MotionGenerator = () => {
           }}
         />
       </section>
-      <section className="p-5">
-        {motion && motion.adinfo ? (
-          <section className="flex flex-col items-center mb-5">
-            <section className="flex gap-2 text-2xl items-center">
-              <span className="text-neutral-500">
-                <IconInfo />
-              </span>
-              {infoslideString}
-            </section>
-            <p className="text-justify max-w-[75vw]">{motion.adinfo}</p>
-          </section>
-        ) : (
-          ""
-        )}
-        <p className="text-2xl md:text-2xl max-w-[85vw] mb-5">
-          &quot;{motion?.motion || ""}&quot;
-        </p>
-        <p className="text-neutral-500">{motion?.source || ""}</p>
+      <hr className="border-b-2 mt-8 mb-8 rounded border-neutral-800 my-2 w-full" />
+      <section className="flex flex-col xl:flex-row items-center">
+        <MotionsFilter onFiltersChange={handleFiltersChange} />
+        <MotionDisplay motion={motion} />
       </section>
     </div>
```

Comment:
Seems fine? I'd think about putting the buttons down so that you don't have to scroll to get to them on mobile & so that you don't get the mental lag interacting with the filters and then suddenly having to click the button at the top.